app/src/main/java/org/oppia/android/app/topic/TopicFragmentPresenter.kt

```diff
@@ -77,15 +79,15 @@ class TopicFragmentPresenter @Inject constructor(
 
   private fun setUpViewPager(viewPager2: ViewPager2, topicId: String, isConfigChanged: Boolean) {
     val adapter =
-      ViewPagerAdapter(fragment, internalProfileId, topicId, storyId, enablePracticeTab)
+      ViewPagerAdapter(fragment, internalProfileId, topicId, storyId, enableExtraTopicTabsUi.value)
     viewPager2.adapter = adapter
     TabLayoutMediator(tabLayout, viewPager2) { tab, position ->
-      val topicTab = TopicTab.getTabForPosition(position, enablePracticeTab)
+      val topicTab = TopicTab.getTabForPosition(position, enableExtraTopicTabsUi.value)
       tab.text = resourceHandler.getStringInLocale(topicTab.tabLabelResId)
       tab.icon = ContextCompat.getDrawable(activity, topicTab.tabIconResId)
     }.attach()
-    if (!isConfigChanged && topicId.isNotEmpty()) {
-      setCurrentTab(if (storyId.isNotEmpty()) TopicTab.LESSONS else TopicTab.INFO)
+    if (!isConfigChanged && topicId.isNotEmpty() && enableExtraTopicTabsUi.value) {
```

Comment:
Is the ``topicId.isNotEmpty()`` still the correct logic here when the parameter is enabled? I suspect there are other test cases that you could add that could cause this to unexpectedly break (e.g. no topic ID but the parameter is enabled).

Reply:
perhaps you meant storyId instead of topicId in this above comment? 
if this^ is correct, what should we do if the parameter is "set" and there is no storyId available?

Reply:
Please reply to this

Reply:
As discussed in the meeting, "storyId" would never be null under normal app scenarios.